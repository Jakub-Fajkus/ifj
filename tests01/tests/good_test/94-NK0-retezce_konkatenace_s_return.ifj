class Main
{
	static void run()
	{
		String str1;
		str1 = Game.play();
		str1 = "Toto je nejaky text " + str1;
		ifj16.print(str1);
	} // end of static void run()
} // end of class Main
class Game
{
	static String play()
	{
		String str1 = "475";
		return str1;
	} // end of static void play(String)
} // end of class Game
