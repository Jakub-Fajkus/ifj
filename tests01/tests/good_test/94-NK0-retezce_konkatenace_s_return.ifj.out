Toto je nejaky text 475